### PR TITLE
Actual requests

### PR DESCRIPTION
**Pull request by malarzm**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -4,15 +4,83 @@
...
+                [
+                    RequestOptions::JSON => \array_merge($auth, $request->toArray()),
+                ]
+            );
+            $responseText = $response->getBody()->getContents();
+        } catch (GuzzleException $e) {
+            throw new ServiceUnavailable($e->getMessage(), $e->getCode(), $e);
+        }
+
+        $decoded = \json_decode($responseText, true);
```
Comment (lukasz-falda) [inline src/Client.php]: Consider to add `JSON_THROW_ON_ERROR` option

File: src/Exception/MalformedResponse.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\FocusContactCenterBundle\Exception;
+
+use Throwable;
```
Comment (lukasz-falda) [inline src/Exception/MalformedResponse.php]: use `\Throwable` and remove this `use`

File: src/HashGenerator.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    private $configuration;
+
+    public function __construct(Configuration $configuration)
+    {
+        $this->configuration = $configuration;
+    }
+
+    public function hash(string $change): string
+    {
+        [$login, ] = explode('@', $this->configuration->getLogin());
```
Comment (lukasz-falda) [inline src/HashGenerator.php]: After you add a DependecyInjection for configuration, please check that you have a `@` at login.

File: src/Response/FCC/AddRecords.php [hunk 4]
```diff
@@ -21,6 +23,24 @@ public function __construct(array $recordsId)
         $this->recordsId = $recordsId;
     }
 
+    public static function fromArray(array $response): self
+    {
+        try {
+            Assert::keyExists($response, 'records_id');
+            Assert::isArray($response, 'records_id');
```
Comment (lukasz-falda) [inline src/Response/FCC/AddRecords.php]: Why you return `records_id` as the exception message? in `isArray` method. Looks like mistake.

File: src/Client.php (hunk 1 above)
Comment (malarzm) [reply src/Client.php]: `JSON_THROW_ON_ERROR` was added in PHP 7.3 while this package has minimum requirement of 7.1

**Pull request merged**